test/test_ops.py

```diff
@@ -160,6 +160,43 @@ def test_numpy_ref(self, device, dtype, op):
         finally:
             torch.set_default_dtype(cur_default)
 
+    # Tests that the cpu and gpu results are consistent
+    @onlyCUDA
+    @suppress_warnings
+    @ops(_ops_and_refs)
```

Comment:
This test should only run for operators that do not have Python references, since there's already a tests that checks for consistency with those

Reply:
I removed all ops where `op.ref is not None`